### PR TITLE
fix(swb): fix integ tests and proxyLambda. Added swbv2-private test c…

### PR DESCRIPTION
**Pull request by kpark277**
…onfig

Issue #, if available:

Description of changes:
* Added swbv2-private integration-test config
* Updated proxyLambdaHandler to properly handle responses from API Gateway
* Updated integ-tests to match return from proxyLambdaHandler

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [x] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts [hunk 1]
```diff
@@ -158,6 +158,7 @@ describe('create environment type configs', () => {
       checkHttpError(
         e,
         new HttpError(403, {
+          statusCode: 403,
```
Comment (aws-tyler) [inline solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts:161]: Why is there a status code here?

File: solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/list.test.ts [hunk 2]
```diff
@@ -42,6 +42,7 @@ describe('list environment type configs', () => {
       checkHttpError(
         e,
         new HttpError(403, {
+          statusCode: 403,
```
Comment (aws-tyler) [inline solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/list.test.ts:45]: Why is there a status code here?

File: solutions/swb-reference/integration-tests/tests/multiStep/userProject.test.ts [hunk 3]
```diff
@@ -46,7 +46,7 @@ describe('multiStep user to project integration test', () => {
       const { data: user } = await adminSession.resources.users.create({
         firstName: 'Project',
         lastName: role,
-        email: `success-user-to-project-user-${uuidv4()}@simulator.amazonses.com`
+        email: `success+user-to-project-user-${uuidv4()}@simulator.amazonses.com`
```
Comment (aws-tyler) [inline solutions/swb-reference/integration-tests/tests/multiStep/userProject.test.ts:49]: why the switch?

**Review by aws-tyler — CHANGES_REQUESTED** [review]
Can we update the response so it doesn't return the http status in the body?

File: solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts (hunk 1 above)
Comment (aws-tyler) [reply solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts:161]: This shouldn't be a 403, it should be a 400 right? Its a missing value that is required, not an unauthorized access

File: solutions/swb-reference/integration-tests/tests/multiStep/userProject.test.ts (hunk 3 above)
Comment (sevu1) [reply solutions/swb-reference/integration-tests/tests/multiStep/userProject.test.ts:49]: - means nothing, while + will redirect emails to `success@simulator.amazonses.com` . I recently got email about large percentage of bouncing emails for cognito users when was running tests, but didn't realize that `success@simulator.amazonses.com` is actually exist

File: solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts (hunk 1 above)
Comment (aws-tyler) [reply solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts:161]: Apparently this is an existing issue. I'm not going to block the PR on it.

**Review by aws-tyler — APPROVED** [review]

**Pull request merged**